### PR TITLE
Indefinite article change for RHOSP abbreviation

### PR DESCRIPTION
**Pull request by maxwelldb**
Additionally:

* Adding OSP to common-atributes

Context: https://github.com/openshift/openshift-docs/pull/17304

For 4.2.

### CODE REVIEW

**Comment by maxwelldb** [comment]
@bergerhoffer Do you want this one?

File: modules/installation-osp-default-deployment.adoc [hunk 1]
```diff
@@ -11,9 +11,9 @@ endif::[]
 [id="installation-osp-default-deployment_{context}"]
 = Resource guidelines for installing {product-title} on OpenStack
 
-Your quota must meet the following requirements to run the {product-title} installation program in RHOSP:
+Your quota must meet the following requirements to run the {product-title} installation program in {rh-openstack}:
```
Comment (bergerhoffer) [inline modules/installation-osp-default-deployment.adoc]: This should probably be the {rh-openstack-first} attribute since it's the first instance in the module.

File: modules/installation-osp-describing-cloud-parameters.adoc [hunk 2]
```diff
@@ -7,15 +7,15 @@
 [id="installation-osp-describing-cloud-parameters_{context}"]
 = Defining parameters for the installation program
 
-The {product-title} installation program relies on a file called `clouds.yaml`. The file describes RHOSP configuration parameters, including the project name, log in information, and authorization service URLs.
+The {product-title} installation program relies on a file called `clouds.yaml`. The file describes {rh-openstack} configuration parameters, including the project name, log in information, and authorization service URLs.
```
Comment (bergerhoffer) [inline modules/installation-osp-describing-cloud-parameters.adoc]: Same here, to use {rh-openstack-first}

File: modules/installation-osp-enabling-kuryr.adoc [hunk 3]
```diff
@@ -9,16 +9,16 @@ You can enable Kuryr in your project by editing a parameter in the `install-conf
 
 .Prerequisites
 
-* Kuryr requires greater than the minimum recommended resources. As an administrator, increase Red Hat OpenStack Platform's (RHOSP) resource quotas:
+* Kuryr requires greater than the minimum recommended resources. As an administrator, increase Red Hat OpenStack Platform's ({rh-openstack}) resource quotas:
```
Comment (bergerhoffer) [inline modules/installation-osp-enabling-kuryr.adoc]: should use {rh-openstack-first} instead of replacing just the RHOSP part with {rh-openstack}.

Oh I see why, because of the "'s". Maybe reword to work around it? "increase the resource quotas for {rh-openstack-first}" ?

File: modules/installation-osp-enabling-swift.adoc [hunk 4]
```diff
@@ -7,20 +7,20 @@
 [id="installation-osp-enabling-swift_{context}"]
 = Enabling Swift on OpenStack
 
-{product-title} on RHOSP uses https://docs.openstack.org/security-guide/object-storage.html[OpenStack Object Storage (Swift)] to store and serve user configuration files.
+{product-title} on {rh-openstack} uses https://docs.openstack.org/security-guide/object-storage.html[OpenStack Object Storage (Swift)] to store and serve user configuration files.
```
Comment (bergerhoffer) [inline modules/installation-osp-enabling-swift.adoc]: Same here, to use {rh-openstack-first}

**Review by bergerhoffer — COMMENTED** [review]
Just a few minor things, otherwise looks good!

File: modules/installation-osp-enabling-kuryr.adoc (hunk 3 above)
Comment (maxwelldb) [reply modules/installation-osp-enabling-kuryr.adoc]: Oh, whoops. Yeah. Fixed! Thanks.

**Comment by maxwelldb** [comment]
@bergerhoffer Think I hit all of your comments. Let me know and I'll autosquash this.

**Comment by bergerhoffer** [comment]
@maxwelldb Yep looks good to me now if you want to go ahead and squash! And then just let me know if you're ready for me to merge after that.

**Comment by maxwelldb** [comment]
Realized I reflexively edited in "Merge away" into that last comment. Just so that's in your inbox, too, hello @bergerhoffer

**Comment by bergerhoffer** [comment]
Appreciate the clarification, merging/picking!
